sdk/resourcemanager/azure-resourcemanager-containerinstance/src/test/java/com/azure/resourcemanager/containerinstance/ContainerGroupTest.java

```diff
@@ -225,4 +228,135 @@ public void testBeginCreateWithFileShareVolume() {
         ContainerGroup containerGroup = acceptedContainerGroup.getSyncPoller().getFinalResult();
         Assertions.assertEquals(1, containerGroup.volumes().size());
     }
+
+    @Test
+    public void testCreateWithDnsNameLabelNoReuse() {
+        String containerGroupName = generateRandomResourceName("container", 20);
+        String dnsNameLbl = generateRandomResourceName("dnslbl", 20);
+
+        Region region = Region.US_WEST3;
+
+        // With NoReuse
+        ContainerGroup containerGroup = containerInstanceManager.containerGroups()
+            .define(containerGroupName)
+            .withRegion(region)
+            .withNewResourceGroup(rgName)
+            .withLinux()
+            .withPublicImageRegistryOnly()
+            .withoutVolume()
+            .withContainerInstance("nginx", 80)
+            .withDnsPrefix(dnsNameLbl)
+            .withAutoGeneratedDomainNameLabelScope(DnsNameLabelReusePolicy.NOREUSE)
+            .create();
+
+        Assertions
+            .assertTrue(DnsNameLabelReusePolicy.NOREUSE.equals(containerGroup.autoGeneratedDomainNameLabelScope()));
+    }
+
+    @Test
+    public void testCreateWithDnsNameLabelUnSecure() {
```

Comment:
What is this test for?

Reply:
Fixed in the [commit#6e748a3](https://github.com/Azure/azure-sdk-for-java/pull/42795/commits/6e748a33351eeda68205b4a03d69db268dd3d0ed)